spec/knapsack_pro/config/env_spec.rb

```diff
@@ -667,38 +653,94 @@
     end
   end
 
-  describe '.fixed_queue_split' do
```

Comment:
This is already tested by https://github.com/KnapsackPro/knapsack_pro-ruby/pull/198/files#diff-e619f118627dce96c5bb61b9e9d7b24d052e0959bc5f95b4b9127d3d76beeb81R656